test/promql/tests/application_errors_test.yaml

```diff
@@ -0,0 +1,108 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.application_alerts.yaml
+
+tests:
+  # Alerted cases:
+  - interval: 5m
+    input_series:
+
+      # HAS is experiencince Application deletion failure rate of 100%, so it will be alerted
+      - series: 'has_application_successful_deletion_total{namespace="application-service", pod="has", source_cluster="cluster01"}'
```

Comment:
No reason in particular, I just added it because it was the metric that HAS would increment in successful scenarios, so I thought it made sense for the tests to use it as well, but since it's not needed, I can remove it.